stable/mysql/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: mysql
-version: 0.12.0
+version: 0.12.1
```

Comment:
This adds a new feature, so could you bump the minor version?

Reply:
ping @imcheck